frontend/src/__tests__/Paragraph.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { ThemeProvider } from 'emotion-theming'
+import theme from '../theme'
+import { Paragraph, P, Lead } from '../components/paragraph'
+
+describe('Paragraph', () => {
+  const example = 'example'
+  const wrapper = (
+    <ThemeProvider theme={theme}>
+      <Paragraph color="black">{example}</Paragraph>
+    </ThemeProvider>
+  )
+
+  it('Renders a Paragraph component, P and Lead presets without crashing', () => {
+    const div = document.createElement('div')
+    ReactDOM.render(wrapper, div)
```

Comment:
Why not use enzyme, for example as in https://github.com/cds-snc/report-a-cybercrime/blob/master/frontend/src/__tests__/Home.test.js#L14-L20

Reply:
Largely because I stole how govuk react does there tests so I could move more quickly, I have no strong feelings on how we test! I'm actually doing some diving into how to test things today